README.md

```diff
@@ -65,3 +65,9 @@ conda env update --file conda_env.yml
 ### Docker installation 
 
 To use `iglu` environment inside the docker container proceed to [Docker installation](docker/getting_started.md) section.
+
+## Known Issues
+
+### Java versions
+
+You might have another java version installed (e.g. `openjdk-11`), rather than `openjdk-8-jdk`. Check the version by running `java --version` or `java -version`.
```

Comment:
To check the version of java, one should use `java -version` (only one hyphen)

Reply:
and to update default java runtime it's either `update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java` or `update-alternatives --config java`. The second one is preferable since it isn't conditioned on a particular path

Reply:
> To check the version of java, one should use `java -version` (only one hyphen)

Actually it's two hyphens for openjdk-11 and one hyphen for openjdk-8. I don't know why but at least in my local machine it was like this.

Reply:
Ok. Resolving.